astro/src/content/blog/fusionauth-funding.mdx

```diff
@@ -0,0 +1,33 @@
+---
+publish_date: 2023-10-31
```

Comment:
You probably want this to be Nov 1, right?

Reply:
Yeah. Thanks. I had to change to today to get it to appear in localhost.